driver-core/src/main/com/mongodb/internal/connection/DefaultConnectionPool.java

```diff
@@ -750,18 +754,6 @@ public <T> T receive(final Decoder<T> decoder, final OperationContext operationC
             return wrapped.receive(decoder, operationContext);
         }
 
-        @Override
```

Comment:
This method has already been removed in https://github.com/mongodb/mongo-java-driver/commit/01128b7c2deea570c9bc2b77671a598ab83d902e#diff-7f8dc15248ba3fe10795342e55c0525fee969cc73d918141a5da85dbded487fb.  Maybe a bad merge?

Reply:
I think this is the merge commit where the method was added back: https://github.com/mongodb/mongo-java-driver/commit/7c5d986374810a0c14400fdb38dc510e375c0ab5#diff-7f8dc15248ba3fe10795342e55c0525fee969cc73d918141a5da85dbded487fb

Reply:
Looks like I messed up the merge and kept the method in:

```
        @Override
<<<<<<< HEAD
        public boolean supportsAdditionalTimeout() {
            isTrue("open", !isClosed.get());
            return wrapped.supportsAdditionalTimeout();
        }

        @Override
        public <T> T receive(final Decoder<T> decoder, final OperationContext operationContext, final int additionalTimeout) {
||||||| 091a01fa1c
        public boolean supportsAdditionalTimeout() {
            isTrue("open", !isClosed.get());
            return wrapped.supportsAdditionalTimeout();
        }

        @Override
        public <T> T receive(final Decoder<T> decoder, final SessionContext sessionContext, final int additionalTimeout) {
=======
        public <T> T receive(final Decoder<T> decoder, final SessionContext sessionContext, final int additionalTimeout) {
>>>>>>> master
```

We'll have to do a review of the final diff of CSOT